### PR TITLE
Update azure-pipelines.yml for Azure Pipelines

### PR DESCRIPTION
**Pull request by ShaunaSong**

### CODE REVIEW

**Review by pavelprystinka — DISMISSED** [review]

**Review by iaulakh — DISMISSED** [review]

File: eng/pipelines/Localization.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+# Secret variable 'tdbSecret' detected in `variables`, follow the guidance on handling secret variables: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables
+# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
+#       env:
+#           MY_ACCESS_TOKEN: $(System.AccessToken)
+# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# No trigger found, defaulting to 'none'. Update the trigger as needed.
+# No name found, setting the default value '$(Date:yyyyMMdd).$(Rev:r)'. This value determines how your pipeline runs are numbered. Update the name as needed: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/run-number?view=azure-devops&tabs=yaml
+# The pipeline uses service connection "ServiceTree Gateway", you will need to grant the new pipeline access: https://dev.azure.com/skype/SCC/_settings/adminservices?resourceId=c29e963a-8ea3-4e6a-b28f-a035eaff83ab
```
Comment (iaulakh) [inline eng/pipelines/Localization.yml:12]: can we remove this url?

File: eng/pipelines/Localization.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# This pipeline was generated from the classic pipeline "SCC-SPOOL-communication-ui-library-Android-Localization" on 2023-10-28 with https://aka.ms/1ESPTMigration (v1.0.0): https://dev.azure.com/skype/SCC/_build?definitionId=12525
```
Comment (iaulakh) [inline eng/pipelines/Localization.yml:1]: can we remove comments as it have many urls?

File: guardian/SDL/.gdnsuppress [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "results": {
+    "7f19827851560b611301d4c44f50a869871f22e6b698d213dc5f8fe6da35c6d2": {
+      "signature": "7f19827851560b611301d4c44f50a869871f22e6b698d213dc5f8fe6da35c6d2",
+      "alternativeSignatures": [],
+      "memberOf": [
+        "default"
+      ],
+      "createdDate": "2023-11-01 19:19:31Z"
+    }
```
Comment (iaulakh) [inline guardian/SDL/.gdnsuppress:23]: is it safe to commit this in public repo?

Comment (ShaunaSong) [reply guardian/SDL/.gdnsuppress:23]: this is auto generate same for all.

**Review by iaulakh — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Pull request merged**